verification/curator-service/api/src/controllers/auth.ts

```diff
@@ -21,12 +21,13 @@ export const mustBeAuthenticated = (
 };
 
 // mustHaveAnyRole is an express middleware that checks that the currently logged-in user has any of the given roles.
-export const mustHaveAnyRole = (requiredRoles: Set<string>) => {
+export const mustHaveAnyRole = (requiredRoles: string[]) => {
     return (req: Request, res: Response, next: NextFunction): void => {
+        const requiredSet = new Set(requiredRoles);
```

Comment:
Is the change from Set\<s> to s[] to simplify the API? That's cool, but it's probably more expensive to construct a set and `.has()` on it for every request than it is to just inspect s[].

Reply:
Yes this was to avoid the callers having to build a Set which felt awkward as an API, the set is actually constructed once and a function is returned that uses it so there's no performance difference.